### ISSUE TITLE
Provide sensible description of deb packages.

### ISSUE THREAD

**Issue by baryluk** [ROCm/ROCK-Kernel-Driver#64]
`rocm-dkms` and `rock-dkms` do have very short and not helpful first line description (a Subject), and no actual description.

```
Description: Radeon Open Compute (ROCm) Runtime software stack
```

```
Description: rock-dkms driver in DKMS format.
```

Both of these are very unhelpful when setting up and installing modules. They lack actual content, (multi line description). First one is basically incorrect. And second one is tautological and provides no useful information at all.

I also believe the "Section" field should be `kernel` for both of them, not `misc` (for `rock-dkms`) or `devel` (for `rocm-dkms`).

**Comment by jlgreathouse** [comment]
`rock-dkms` is a package that contains the DKMS-based versions of our `amdgpu`, `amdkfd`, and other GPU drivers that have modifications to allow ROCm to work. The actual code for the drivers that are in this package are contained in this repo, but the package ships a version of this code that is configured as a DKMS module. Note that you may not need to install `rock-dkms` to enable ROCM, as we have been pushing most of these changes into the upstream Linux kernel.

`rocm-dkms` is a meta-package that will install the `rock-dkms` driver package and the `rocm-dev` user-level software package. The latter is another meta-package which will install various ROCm software utilities.

I agree that the Sectiosn for `rock-dkms` should likely be `kernel`. I'm less certain about `rocm-dkms` since it also provides "[Development utilities, compilers, development environments, libraries, etc.](https://packages.debian.org/unstable/)". The discussion about the `rocm-dkms` package should probably take place in an issue on [the main project repo](https://github.com/RadeonOpenCompute/ROCm), however.

@fxkamd -- what are your thoughts on updating the `rock-dkms` .deb information per this request?

**Comment by fxkamd** [comment]
Sure. Updating the description wouldn't hurt. We should coordinate this with the team that builds the amdgpu-dkms packages, because they are built using basically the same scripts.

**Comment by kentrussell** [comment]
Addressed in ROCm 4.5
